language/stdlib/modules/LibraAccount.move

```diff
@@ -276,6 +276,10 @@ module LibraAccount {
         deposit(payer_address, payee_address, coin1, x"", x"");
         deposit(payer_address, payee_address, coin2, x"", x"")
     }
+    spec fun unstaple_lbr {
+        /// > TODO: timeout
+        pragma verify = false;
+    }
```

Comment:
DId this verify with a larger timeout? (Or perhaps did verify locally for you but not in the bot?)

In this case I recommend to use `pragma verify_duration_estimate = 100`. This will make this function excluded from verification if the timeout is not >= 100s (the default is 40). Using this pragma allows us to distinguish cases which do verify but just not in bounds from cases we do not known whether they even verify.

@emmazzz @shazqadeer @DavidLDill for this advice FYI